$OpenBSD: patch-evutil_rand_c,v 1.2 2013/11/11 21:54:57 landry Exp $
OpenBSD does not need nor provide arc4random_addrandom anymore.
And evutil_secure_rng_add_bytes is a bad API anyway.
--- evutil_rand.c.orig	Thu Aug  2 17:36:53 2012
+++ evutil_rand.c	Mon Nov 11 22:41:15 2013
@@ -150,10 +150,3 @@ evutil_secure_rng_get_bytes(void *buf, size_t n)
 	ev_arc4random_buf(buf, n);
 }
 
-void
-evutil_secure_rng_add_bytes(const char *buf, size_t n)
-{
-	arc4random_addrandom((unsigned char*)buf,
-	    n>(size_t)INT_MAX ? INT_MAX : (int)n);
-}
-
